config/new_relic_otlp/README.md

```diff
@@ -0,0 +1,32 @@
+# New Relic OTLP Exporter Prerequisites
+
+New Relic supports native ingestion of OTLP telemetry. The [Open Telemetry OTLP gRPC exporter](https://github.com/open-telemetry/opentelemetry-collector/tree/main/exporter/otlpexporter) can be used to send metrics, logs, and traces to New Relic.
```

Comment:
This link isn't locked to the version of OTLP we support. Not sure if that's an issue or not. Just want to bring it up.

Reply:
I was thinking about this, and I decided to link directly to the exporter doc because the otlp exporter **should** be stable.